commands/en/tos.js

```diff
@@ -3,10 +3,29 @@
 module.exports = {
   name: 'tos',
   category: 'utility',
-  description: '[Mandarin Only] Terms Of Services',
+  description: 'Terms of Service for this bot, please be sure to abide by it.',
   aliases: [],
   run: async (bot, msg, args) => {
-    // 如果不想顯示該指令 請刪掉該檔案 owo
-    return msg.channel.send(':x: **Sorry, the command hasn\'t been translated yet!**')
+    // Delete this file if you dont want this command ;w;
+    const { config, MessageEmbed } = bot
+    return msg.channel.send({
+      embeds:
+        [
+          new MessageEmbed()
+            .setTitle('v6 Music Bot | Terms of Service')
```

Comment:
Recommend: Replace v6 Music Bot with Kristen v6

```suggestion
            .setTitle('Kristen v6 | Terms of Service')
```